src/commonMain/kotlin/indexer/models/chain/OnChainDelegationResponse.kt

```diff
@@ -0,0 +1,23 @@
+package indexer.models.chain
+
+import kotlinx.serialization.Serializable
+
+@Serializable
```

Comment:
[q] do objects not natively come with `.toString()` in abacus? what does `@Serializable` actually do?

Reply:
@Serializable is needed to decode the json string to the data class.